DashWallet/Sources/UI/Explore Dash/Views/DWExploreTestnetContentsView.m

```diff
@@ -218,7 +228,8 @@ - (void)configureHierarchy {
                                                constant:-4],
         [stackView.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor
                                                 constant:15],
-        [stackView.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor]
+        [stackView.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor],
+        [stackView.heightAnchor constraintGreaterThanOrEqualToConstant:64.0]
```

Comment:
This plus `UITableViewAutomaticDimension` seems to work well for adjusting the cell height to the content.